cloud-pipeline-common/model/src/main/java/com/epam/pipeline/vo/filter/FilterExpressionTypeVO.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2017-2019 EPAM Systems, Inc. (https://www.epam.com/)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.epam.pipeline.vo.filter;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public enum FilterExpressionTypeVO {
+    LOGICAL(0),
+    AND(1),
+    OR(2);
+
+    private long id;
+    private static Map<Long, FilterExpressionTypeVO> idMap = new HashMap<>();
+
+    static {
```

Comment:
do we need this maps in common module to be able to send request.

Reply:
fixed, also removed "namesMap"